docs/source/using_the_ve/virtual_ecosystem_in_static_mode.md

```diff
@@ -0,0 +1,440 @@
+---
+execution:
+  timeout: 90
+jupytext:
+  formats: md:myst
+  main_language: python
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+language_info:
+  codemirror_mode:
+    name: ipython
+    version: 3
+  file_extension: .py
+  mimetype: text/x-python
+  name: python
+  nbconvert_exporter: python
+  pygments_lexer: ipython3
+  version: 3.11.14
+---
+
+# Running the Virtual Ecosystem in Static Mode
+
+## Why use Static Mode?
+
+Imagine standing in a rainforest clearing at dawn. The sun rises, mist lifts, and the
+forest slowly comes to life.
+
+Now imagine you could **pause parts of this scene** — hold
+the soil moisture constant, freeze the behaviour of animals, or keep the microclimate
+unchanged — while the rest of the ecosystem continues to move through time.
+
+This is the essence of the **Static Mode** in the Virtual Ecosystem model.
+
+In complex systems where **everything interacts** it can be hard to isolate what’s
+causing what. **Static Mode helps untangle this web** by letting you "freeze" components
+like plants or animals while others remain dynamic. At each model step, static
+components reset to their original state (e.g. eaten leaves magically reappear) —
+like a controlled reset — so any change comes only from the active, evolving parts.
+
+This setup enables **targeted experiments** within the full model framework. For
+example, you can test how vegetation responds to changing temperatures without feedback
+from soil moisture changes, or explore animal behaviour in a fixed vegetation landscape.
+
+By choosing which components evolve over time and which stay fixed, Static Mode helps
+reveal **drivers of ecological change** - helping us understand not just
+**what happens**, but **why**.
```

Comment:
I like the writing style here but it makes very heavy use of bold, which is not recommended for accessibility in screen text. I think the key phrase here is "targeted experiments" - I would genuinely only bold that.

The same is true below - only use bold sparingly, not for every line in a list.